### DISCUSSION TITLE
HA ZHA su dispositivo remoto

### DISCUSSION THREAD

**Discussion by issuluca** [indomus/forum#594] [HUB: Home Assistant]
E' possibile installare ZHA su un dispositivo raspberry e gestirlo da home assistant installato su altro dispositivo server (minipc)
Vorrei sistemare il server di home assistant in un punto della casa, e la chiavetta zigbee al centro della casa collegato ad un raspberry pi3 che non utilizzo.
E' possibile?

**Comment by snakuzzo** [comment]
Dovresti prima dire che tipo di chiavetta hai.
Ad ogni modo, no...ZHA fa parte di Home Assistant. Non è che andrai ad installare ZHA su altro server. Quello che puoi fare è far comunicare ZHA (che risiede sul tuo server) con una chiavetta remota.

Ad esempio se usi la chiavetta [Z-Bee Duo con modulino WIFI per ser2net](https://indomus.it/progetti/adottare-ser2net-ed-esp-01-su-z-bee-duo-per-gestione-remota-con-zigbee2mqtt/) di @gio-dot dovresti riuscire a posizionare la chiavetta dove ti pare e comunicarci direttamente via TPC.
Su ZHA a quel punto dovrebbe essere necessario (una volta selezionato il chip zigbee relativo alla tua chiavetta) indicare come indirizzo `socket://indirizzo_ip:porta`

Se non hai la chiavetta che ti ho menzionato, puoi farlo lo stesso collegando ad esempio la tua cihavetta zigbee ad un RPI Zero W sul quali avrai RaspOS e sul quale andrai ad installare ser2net

**Comment by gio-dot** [comment]
Confermo quanto detto da @snakuzzo. Ho testato la mia chiavetta (https://www.tindie.com/products/gio_dot/z-bee-duo-hybrid-cc2652p-zigbee-30-adapter/) con zha in entrambe le configurazioni usb o ser2net. Per la ser2net va impostato il percorso che si vede in figura. Passando da utilizzo usb a utilizzo ser2net si deve cancellare l'integrazione del coordinator usb e ricrearla di nuovo con il percorso 'socket://192.168.1.xxx:23'. L'accoppiamento con i dispositivi precedenti NON viene perso.


![2021-08-08 11_18_25-Impostazioni - Home Assistant](https://user-images.githubusercontent.com/55327160/128627558-86872815-5a37-430d-a19b-d624696048a6.png)
